src/test/regress/expected/local_shard_copy.out

```diff
@@ -28,9 +28,11 @@ SELECT create_distributed_table('distributed_table','key');
 (1 row)
 
 INSERT INTO distributed_table SELECT *,* FROM generate_series(20, 40);
+DEBUG:  Creating router plan
```

Comment:
We have lots of new `Creating router plan` logs. I was not sure whether we should consider omitting it.

But I decided to keep these are it is really what happens underlying.